test/gtest/shp/containers.cpp

```diff
@@ -26,6 +26,22 @@ TYPED_TEST(DistributedVectorTest,
   EXPECT_EQ(second[0], lib::ranges::segments(second)[0][0]);
 }
 
+TYPED_TEST(DistributedVectorTest, fill_constructor) {
+  EXPECT_TRUE(equal(typename TestFixture::DistVec(10, 1),
+                    typename TestFixture::LocalVec(10, 1)));
+}
+
+// TODO: add more tests: fill_constructor_one_item, fill_constructor_no_items
```

Comment:
@haichangsi , as an exercise of writing concise UTs please implement this TODO and TODO below